header.php

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+ 
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+
+  <title>123 Contact Form Test</title>
+ 
+  <script src="library/jquery-3.2.1.min.js"></script>
```

Comment:
Pentru a imbunatati timpii de incarcare ai paginii, in <head> nu se include javascript-ul, ci exact inainte de terminarea nodului <body>